Product Approval Process + Merchant Context (#763)
spryker/spryker-docs: docs/marketplace/user/merchant-portal-user-guides/202204.0/products/abstract-products/managing-marketplace-abstract-product.md

## Prerequisites

To start working with marketplace abstract products, go to the **Merchant Portal** > **Products**.

> This article contains reference information. Make sure to review it before you start, or just look up the necessary information as you go through the process.

Reviewer: Please consider revering to "document" and without "just"

The following drop-down menus can be used to filter the marketplace abstract products on the *Products* page:

- Categories where the abstract product belongs.
- Visibility (defines whether the product is online or offline).
- Stores where the abstract product is available.
- Approval status (defines whether the product is approved or denied by the marketplace administrator).

The page refreshes and displays the available options as soon as the filter parameters are selected.

To sort the existing abstract products, select one or more sorting parameters from the drop-down menu on the *Products* page. Choose among the following sort criteria:

- SKU
- Name
- Number of variants
- Visibility status
- Approval status

By default, the table is sorted descendingly by SKU.

## Editing abstract product details

To edit the existing marketplace abstract product:

1. Next to the abstract product, you want to edit, hover over the three dots and click **Manage Product** or just click the line. This takes you to the *[Product name]*, *Abstract Product Details* tab.
2. In the *Name & Description* pane, edit *Name* and *Description* for every locale.

> 3. In the *Stores* pane, in the drop-down menu select the stores where the product is available.

Reviewer: ```suggestion
3. In the *Stores* pane, in the drop-down menu, select the stores where the product is available.
```

| Attributes  | In this pane, you can manage attributes for your product. See [Managing abstract product attributes](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-attributes.html) for more details. |           |
| Categories  | Defines the [categories](/docs/scos/user/features/{{page.version}}/category-management-feature-overview.html) the product is displayed in. |           |
| SEO         | In this pane, you can manage meta information for your product. See [Managing abstract product meta information](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/managing-marketplace-abstract-product-meta-information.html) for more details. |           |

## Sending the product for approval

{% info_block warningBox "Warning" %}

You can approve only the [newly created marketplace product](/docs/marketplace/user/merchant-portal-user-guides/{{page.version}}/products/abstract-products/creating-marketplace-abstract-product.html).

{% endinfo_block %}

> In order for the new product to be available on the Storefront, it needs to be approved. To send the product for approval:

Reviewer: ```suggestion
For the new product to be available on the Storefront, it needs to be approved. To send the product for approval, do the following:
```
